pom.xml

```diff
@@ -134,15 +121,6 @@
             </snapshots>
         </pluginRepository>
 
```

Comment:
Please remove the empty line as well. Ty.

Reply:
fixed